docs/components/early-access/alpha/mcp-client/mcp-tool-discovery.md

```diff
@@ -0,0 +1,91 @@
+---
+id: mcp-client-tool-discovery
+title: MCP Client tool discovery
+sidebar_label: Tool discovery
+description: "Learn how AI agents use gateway tool definitions to automatically discover and invoke MCP client tools."
+---
+
+Learn how AI agents use gateway tool definitions to automatically discover and invoke MCP client tools, enabling distributed, advanced BPMN workflows.
+
+## About
+
+AI agents can detect and use tools provided by MCP clients via activities within an ad-hoc sub-process. This is made possible by so-called _gateway tool definitions_, which provide access to a list of tools instead of a single tool definition.
```

Comment:
We recently added a short section on those gateway tool definitions where we could link to: https://preview.docs.camunda.cloud/pr-7529/docs/next/components/connectors/out-of-the-box-connectors/agentic-ai-aiagent-tool-definitions/#gateway-tool-definitions